index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Personal Website</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="index.js" defer></script>
+</head>
+<body>
+    <!-- <button id="darkBtn" onclick="darkMode()">Dark Mode</button> -->
+    <button onclick="topFunction()" id="topBtn" title="Go to top">Back to Top</button>
+    <div class="grid-container">
+        <div class="card1">
+    <section id="home">
+        <h1>Flower Aviles</h1>
+        <nav>
+            <ul>
+                <li> <a class="navigation-links" href="#home">Home</a></li>
+                <li> <a class="navigation-links" href="#aboutMe">About Me</a></li>
+                <li> <a class="navigation-links" href="#projects">Projects</a></li>
+                <li> <a class="navigation-links" href="#resume">Resume</a></li>
+                <li> <a class="navigation-links" href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </section>
+        </div>
+
+        <div class="card">
+    <section id="aboutMe">
+        <h2>About Me</h2>
+        <img class="imgOfMe"src="Flower.JPG" alt="image of Flower">
+        <p>
+            I have had quite the adventure getting to where I am today. I can confidently say that I have no regrets, because every hardship that i have endured, created the person that I am today.
+            <span id="dots">...</span>
+            <span id="more">Along the way I found my purpose and that is becoming the greatest asset in the tech industry. I still have a long way to go and the people that i have met in this journey have always added new perspectives to my life, which some have become my closest confidants. I am an introvert, however I love to help everyone as much as time would allow me to. I love to sing and play the piano so much that once upon a time I wanted to pursue a music career. Now I have grown to coding and building programs that one day could be helpful to others.</span>
+        </p>
+        <button onclick="readMore()" id="readBtn">Read more</button>
```

Comment:
We should no longer be using inline function calls. Instead grab the button and add an event to fire this function in you JS file.